src/elf.rs

```diff
@@ -20,12 +20,20 @@ use {
 };
 
 /// An ELF object file.
-#[derive(Debug)]
 pub struct ElfFile<'data> {
     elf: elf::Elf<'data>,
     data: &'data [u8],
 }
 
+impl<'data> fmt::Debug for ElfFile<'data> {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        fmt.debug_struct("ElfFile")
+            .field("elf", &self.elf)
+            .field("data", &"<... redacted ...>")
```

Comment:
I'm leaning towards wrapping `data` in a newtype that does this instead. This also has the added benefit that the newtype can enforce bounds checks on access to the data (eg to fix #32).

Reply:
I see. Here is the script to check all `[u8]` buffer: `git grep -n -E "&('\w+ )?\[u8\]"`.